app/src/main/java/fr/free/nrw/commons/upload/FileProcessor.kt

```diff
@@ -82,6 +84,12 @@ class FileProcessor @Inject constructor(
             defaultKvStore.getStringSet(Prefs.MANAGED_EXIF_TAGS) ?: emptySet()
         val redactTags: Set<String> =
             context.resources.getStringArray(R.array.pref_exifTag_values).toSet()
+        /* Remove EXIF location if user has chosen
```

Comment:
If the picture itself already has EXIF, we should not remove it.
Though this can be kept as commented code, with a note saying `// This will be useful when implementing https://github.com/commons-app/apps-android-commons/issues/5247`.

Reply:
Done.